packages/table/src/table.tsx

```diff
@@ -1518,11 +1553,9 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
                 style.left = "-1px";
                 style.borderLeft = "none";
                 style.bottom = "-1px";
-                style.transform = `translate3d(${-viewportRect.left}px, 0, 0)`;
```

Comment:
Delete these transforms to prevent header selection overlays from scrolling twice as far as they should.